### ISSUE TITLE
Update transaction ID shown to the user

### ISSUE THREAD

**Issue by vadaynujra** [pendulum-chain/vortex#193]
### Context
The current transaction ID is a SEP24 generated ID and isn't captured in the storage file. 

### Requirement

- [ ] Update transaction ID to use the last 12 digits of the transaction hash on Polygon (for the token approval transaction)
- [ ] Capture in storage file
- [ ] Display to the user in 3 groups of 4 characters

**Comment by TorstenStueber** [comment]
@vadaynujra I updated the description. We already store the complete id in the sheet so there is no work required for additional data dumping.

**Comment by prayagd** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/vortex/193?planning-poker) with Zenhub @b-yap @bogdanS98 @ebma @gianfra-t @TorstenStueber

**Comment by prayagd** [comment]
Remove it from success page and keep it on unsuccess page

**Comment by prayagd** [comment]
@vadaynujra as discussed the `squidRouterReceiverId` is of no use to the end user, we would need to define what exactly to be shown here

@TorstenStueber i saw after the bridge(polygon to moonbeam) is completed there is txn hash printed in the console but didnt find that hash in the dumping sheet. IMO showing that hash to the end user would be somewhat helpful

**Comment by ebma** [comment]
>  the `squidRouterReceiverId` is of no use to the end user, we would need to define what exactly to be shown here

Maybe we should discuss this again and with more context. What exactly is the need for this change? Why would we want to show a different ID to the user? Is it a) to be able to assist with debugging if the user approaches us and tells us the exact ID shown to them on the error page or b) the user is able to investigate the issue by themselves? I understood it's about a) and thus, showing the `squidRouterReceiverId` would be fine as it helps us internally.

**Comment by TorstenStueber** [comment]
Yes, I totally agree with @ebma, that is also my understanding. We also don't link to explorers that show any of the many other transactions, so why do it for this very first step of the flow?

**Comment by ebma** [comment]
@pendulum-chain/product we did not find an agreement here. Please elaborate on what this identifier is for so we can judge whether it's really needed or how it should look to achieve what you want to gain from it.

**Comment by vadaynujra** [comment]
I consider this a Developer user story, to improve / simplify debugging internally. Based on that let's go with the suggestion of squidrouterreceiverID which I see is already updated in the description.

**Comment by vadaynujra** [comment]
@ebma with the above solution suggestion, how would we debug transactions where the user is sent to the unsuccessful screen without signing both transactions?

**Comment by TorstenStueber** [comment]
We create this id already at the moment the user starts the anchor flow. Failures can't happen before anyway. This means that we will always have the id available whenever the failure screen is shown.
